### PR TITLE
implement percent retained

### PR DESCRIPTION
**Pull request by msultan**
Implements the ```percent_retained_``` attribute for MSM as raised in #659

### CODE REVIEW

**Comment by rmcgibbo** [comment]
Nice!

File: msmbuilder/tests/test_strongly_connected_subgraph.py [hunk 1]
```diff
@@ -4,57 +4,59 @@
...
 def test_2():
     C = np.array([[1,1,0],
                   [0,1,1],
                   [0,1,1]])
-    tC, m = _strongly_connected_subgraph(np.array(C))
+    tC, m, p_r = _strongly_connected_subgraph(np.array(C))
     np.testing.assert_array_equal(tC, np.array([[1,1], [1,1]]))
     assert m == {1: 0, 2: 1}
-
+    assert  p_r ==83.333333333333329
```
Comment (rmcgibbo) [inline msmbuilder/tests/test_strongly_connected_subgraph.py]: don't assert on floating point equality. use `np.testing.assert_almost_equal`

Comment (msultan) [inline msmbuilder/tests/test_strongly_connected_subgraph.py]: k, let me make the changes and push again. Thanks!

Comment (rmcgibbo) [inline msmbuilder/tests/test_strongly_connected_subgraph.py]: (see e.g. http://yaroslavvb.blogspot.com/2011/09/dont-use-exact-equality-with-floating.html)

**Pull request merged**